Sources/Twift+Lists.swift

```diff
@@ -103,6 +103,26 @@ extension Twift {
   public func deleteList(_ listId: List.ID) async throws -> TwitterAPIData<DeleteResponse> {
     return try await call(route: .list(listId), method: .DELETE, expectedReturnType: TwitterAPIData.self)
   }
+    
+    /// Enables the authenticated user to create a new List.
+    /// - Parameters:
+    ///  - name: List name (required)
+    ///  - description: Description for the list (optional)
+    ///  - private: true or false (optional)
```

Comment:
This is a bit of a nit, but “true or false” is not a super helpful comment when people can see the accepted type is `Bool`. Try to describe the meaning of the field, e.g. `Whether the list is private to the authenticated user or publicly viewable`

Reply:
I am not used to writing these types of comments but I did pause for a second and thought if I need more context here.

It so makes sense not to mention true/false if they already see a type. 
I hope to get better at this 🤞